examples/01_plotting/plot_surf_atlas.py

```diff
@@ -42,24 +42,21 @@
 fsaverage_meshes = load_fsaverage()
 
 # The fsaverage meshes contains the FileMesh objects:
-print(
-    "Fsaverage5 pial surface of left hemisphere is: "
-    f"{fsaverage_meshes['pial'].parts['left']}"
-)
-print(
-    "Fsaverage5 inflated surface of left hemisphere is: "
-    f"{fsaverage_meshes['inflated'].parts['left']}"
-)
+print(f"{fsaverage_meshes['pial'].parts['left']=}")
+print(f"{fsaverage_meshes['inflated'].parts['left']=}")
```

Comment:
I was not aware of that.

Reply:
this is a convenient little feature of f-strings: very useful when print debugging